packages/core/src/extensions/Blocks/nodes/BlockContainer.ts

```diff
@@ -89,11 +86,16 @@ export const BlockContainer = Node.create<IBlock>({
       }),
       [
         "div",
-        mergeAttributes(HTMLAttributes, {
-          // TODO: maybe remove html attributes from inner block
-          class: styles.block,
-          "data-node-type": this.name,
-        }),
+        mergeAttributes(
+          {
+            ...domAttributes,
+            class: domAttributes.class
```

Comment:
let's start using either https://www.npmjs.com/package/classnames for concatenation, or create a simple helper function ourselves